.github/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to the AtlasOS docs contributing guide
+# Welcome to the AtlasOS documentation contributing guide
```

Comment:
Maybe the contribution guide here could be stripped down (with only repo-specific stuff) after https://github.com/Atlas-OS/docs/pull/76 is merged?

Reply:
I think @JayXTQ or @PencilNavigator would be the best to answer anything regarding the contribution guide within this repository for the documentation.

However, I agree that this repository's contribution guide should contain more information about how to set up a development environment and so forth. Just like #76 does for the Atlas playbook.

Maybe leave this question for an internal discussion for another PR where that might occur?6

Reply:
I made that guide before we even did the internal chat, we can modify it how we wish.

Reply:
@he3als what suggestions do you have then for change? Maybe you'd like to turn it into an internal discussion with @JayXTQ for now? I feel like that change is quite unfitting for this PR.

Reply:
I agree, for now, no changes. Once this is merged, I might include some changes in https://github.com/Atlas-OS/docs/pull/76 and see what you think about it.